packages/codemods/src/codemods/v0.48.x/upgradeYarn/__testfixtures__/default/output/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "engines": {
     "node": ">=14.17 <=16.x",
-    "yarn": ">=1.15 <2"
+    "yarn": ">=3"
```

Comment:
```suggestion
    "yarn": ">=1.15"
```

In the create redwood app template, now using Yarn 3 at the start, we've decided to set it to this. It's a bit subjective, but here's our reasoning:
- Yarn 3 ignores engines
- the locally installed yarn is always v1, which we need to be >= 1.15
- in general, there's a lot of confusion about how these engines work and when they apply

We don't want to make a change based on our settings for Renovate — this is a customization devs would choose on their own and once they know what they're doing may then choose to pin or remove entirely.

All that said, I've never been a fan of engines in general and feel like we should create our own logic. Someday.